src/Context/ComponentOne.js

```diff
@@ -0,0 +1,15 @@
+import { ComponentTwo } from "./ComponentTwo";
+import { UserContext } from './Context';
+import { useState } from 'react';
+
+export const ComponentOne = () =>{
+    const [val, setVal] = useState("I'm being shared")
```

Comment:
state of the application should not be wrapped inside a component!! It should be moved to Context file.  Check the example I shared.

Reply:
done